packages/component-library/package.json

```diff
@@ -20,8 +21,13 @@
   },
   "dependencies": {
     "@quasar/extras": "^1.14.1",
+    "echarts": "^5.3.3",
     "quasar": "^2.7.3",
-    "vue": "^3.2.37"
+    "ramda": "^0.28.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
Do these really need to be dependencies ?

Reply:
We might have them be devDependencies